classes/services/queryBuilders/SubmissionListQueryBuilder.inc.php

```diff
@@ -44,6 +44,38 @@ public function filterBySections($sectionIds) {
 	 * @return object Query object
 	 */
 	public function appGet($q) {
+		$primaryLocale = \AppLocale::getPrimaryLocale();
+		$locale = \AppLocale::getLocale();
+
+		$this->columns[] = Capsule::raw('COALESCE(stl.setting_value, stpl.setting_value) AS section_title');
```

Comment:
Let's talk about this during the dev call. Everywhere where we need to split SQL construction between classes (most typically app/pkp-lib) it's currently spaghetti, so this is an improvement -- but it might be nice to find a way to establish this new pattern with less use of the `raw` construct.